src/agent/pythd/adapter.rs

```diff
@@ -423,7 +426,7 @@ impl Adapter {
             publisher_accounts: price_account
                 .comp
                 .iter()
-                .filter(|comp| **comp != PriceComp::default())
+                .filter(|comp| *comp != &PriceComp::default())
```

Comment:
Lack of this check is not harmful but having a default publisher in logs is annoying.